### PR TITLE
Add Azure Blob Storage Delete operation doc

### PR DESCRIPTION
**Pull request by fabistb**
**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
Adds documentation for the delete operation of the Azure Blob binding.

## Issue reference
#1277

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/blobstorage.md [hunk 1]
```diff
@@ -201,6 +202,82 @@ To perform a get blob operation, invoke the Azure Blob Storage binding with a `P
 
 The response body contains the value stored in the blob object.
 
+### Delete Blob
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-bindings/blobstorage.md]:
```suggestion
### Delete blob
```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]
Since this functionality won't be released until Dapr v1.2, please retarget PR to the v1.2 branch

File: daprdocs/content/en/reference/components-reference/supported-bindings/blobstorage.md (hunk 1 above)
Comment (fabistb) [reply daprdocs/content/en/reference/components-reference/supported-bindings/blobstorage.md]: Updated the file according to your suggestion.

**Comment by fabistb** [comment]
> Since this functionality won't be released until Dapr v1.2, please retarget PR to the v1.2 branch

Unfortunately it slipped through creating the necessary docs PR.
The functionality should already be available in version 1.1.0.

https://github.com/dapr/dapr/releases/tag/v1.1.0
https://github.com/dapr/components-contrib/issues/681

**Comment by AaronCrawfis** [comment]
> > Since this functionality won't be released until Dapr v1.2, please retarget PR to the v1.2 branch
> 
> Unfortunately it slipped through creating the necessary docs PR.
> The functionality should already be available in version 1.1.0.
> 
> https://github.com/dapr/dapr/releases/tag/v1.1.0
> [dapr/components-contrib#681](https://github.com/dapr/components-contrib/issues/681)

Sounds good, thanks for the clarification!

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**